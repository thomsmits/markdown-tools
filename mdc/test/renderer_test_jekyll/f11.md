# Arrows
## Arrows

(-> Arrow) (<- Arrow) (=> darrow) (<= darrow) (<=> DArrow) (<-> darrow)

-> Arrow <- Arrow => darrow <= darrow <=> DArrow <-> darrow

-> Arrow<br><- Arrow<br>=> darrow<br><= darrow<br><=> DArrow<br><-> darrow
