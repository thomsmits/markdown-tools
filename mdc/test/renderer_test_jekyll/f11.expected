---
layout: page
parent: Section 3
title: Arrows
nav_order: 0
---
<h1 class="fw-700 text-purple-300">Arrows</h1>
## Arrows

<p>&rarr; Arrow &larr; Arrow &rArr; darrow &lArr; darrow &hArr; DArrow &harr; darrow
</p>


<p>&rarr; Arrow<br>&larr; Arrow<br>&rArr; darrow<br>&lArr; darrow<br>&hArr; DArrow<br>&harr; darrow
</p>

